app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -48,6 +49,7 @@ class SelectionInteractionViewModel(
       ?: minAllowableSelectionCount
   }
   private val selectedItems: MutableList<Int> = mutableListOf()
+  val selectedItemsCount = ObservableInt()
```

Comment:
Please add a KDoc explaining what this value contains & when it changes. We don't have great KDoc coverage in view models, but we ought to try to moving forward (& especially now that we have a KDoc CI checker that's expecting full documentation coverage for all public API-facing methods & fields).

Reply:
@BenHenning I have added the KDoc for the variable.